### ISSUE TITLE
TS2351: This expression is not constructable.   Type 'typeof import("date-holidays")' has no construct signatures.

### ISSUE THREAD

**Issue by jp928** [commenthol/date-holidays#243]
Typescript version: 4.2.4
date-holiday version: 3.4.0
How to replicate:
```
import * as Holidays from 'date-holidays';

const hd = new Holidays('AU');
```

**Comment by commenthol** [comment]
The interface is 
```js
import Holidays from 'date-holidays'
```

**Comment by jp928** [comment]
@commenthol 
Thanks for your reply.

I had followed your instruction but I got
```
date_holidays_1.default is not a constructor
```

**Comment by commenthol** [comment]
Hi @jp928, 
I hope the issue was found and has been corrected with #245. 
Sorry for the confusion, but thanks to @Nirostar  sample project I could identify the issue.
